tests/test_contact_tools.py

```
@@ -113,18 +116,20 @@ def test_list_contacts_customers(self, mock_zoho_api):
         assert len(result["contacts"]) == 1
         assert result["contacts"][0]["contact_type"] == "customer"
     
-    def test_list_contacts_with_pagination(self, mock_zoho_api):
+    @pytest.mark.asyncio
+    async def test_list_contacts_with_pagination(self, mock_zoho_api):
         """Test listing contacts with pagination parameters."""
         mock_zoho_api.return_value = MOCK_CONTACTS_LIST
         
-        result = list_contacts(page=2, page_size=10)
+        result = await list_contacts(page=2, page_size=10)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)